cpg-language-cxx/src/test/kotlin/de/fraunhofer/aisec/cpg/passes/CallResolverTest.kt

```diff
@@ -717,6 +717,7 @@ class CallResolverTest : BaseTest() {
                 true
             ) {
                 it.registerLanguage<CPPLanguage>()
+                it.inferenceConfiguration(InferenceConfiguration.builder().enabled(false).build())
```

Comment:
inference must be enable because otherwise the invokes edge wont be empty. Personally, I have no idea why it was empty before, I think that actually before the test or this functionality was broken and therefore the invokes edge was empty.